source/assets/stylesheets/_landing.scss

```diff
@@ -125,6 +211,11 @@
       margin-right: 1em;
       padding: $small-spacing 0.5em;
       width: 65%;
+
+      @include media($small-screen) {
+        margin-bottom: $small-spacing;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets